docs/fundamentals/toc.yml

```diff
@@ -152,7 +152,9 @@ items:
     - name: Overview
       href: ../core/sdk.md
     - name: Error messages
-      href: ../core/tools/sdk-errors/netsdk1145.md
+      items:
+      - name: NETSDK1145
```

Comment:
We could, but this will eventually contain many entries, and I expect people will find the one they want by search or by F1 or by filtering on the error number rather than by filtering on "error messages".